test/extern/unions/externUnionNoTypedef.chpl

```diff
@@ -0,0 +1,32 @@
+/**********************************************************
+/* This union example is also provided in
+/* $CHPL_HOME/doc/rst/language/spec/interoperability.rst
+/**********************************************************/
```

Comment:
When I run this, I get:

```
> externUnionNoTypedef.chpl:1: unterminated comment started here
> externUnionNoTypedef.chpl:4: nested comment started here
> externUnionNoTypedef.chpl:33: EOF in comment
```

Maybe you tested this and then added (or changed) the comment afterwards?  Note that Chapel supports nested comments, so each `/*` will require its own matching `*/` to get back out to the code scope again.  If I fix this comment, the test seems to work as expected.

Reply:
Sorry for the carelessness, the habits from C got me. I'll fix it.